addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -1514,6 +1514,25 @@ def test_message_process_file_encoding_ascii(self):
         # This explains the multiple "�" in the attachment.
         self.assertIn("Chauss������e de Bruxelles", record.message_main_attachment_id.raw.decode())
 
+    def test_message_route_reply_model_none(self):
+        message = self.env['mail.message'].create({
+            'email_from': self.email_from,
+            'body': '<p>test</p>',
+            'message_type': 'email',
+            'model': None,
+            'res_id': None,
+        })
+
+        self.env['mail.alias'].create({'alias_name': 'test', 'alias_model_id': self.env['ir.model']._get('res.partner').id})
```

Comment:
Why res_partner ? in test_mail we have a lot of test models, notably "mail.test.gateway", would be better than using a real functional model. Purpose of tests modules is to be independent from standard modules, so that we are not annoyed by functional choices for "frameworky" addons.